### DISCUSSION TITLE
Çöl kaşifleri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#257] [Zeka Sorusu]
Sitenin aktif üyelerinden 4 kaşif çölde inceleme yapmak üzere yola çıkar. Her biri yanında en fazla beş günlük yiyecek taşıyabilmektedir. Kaşifler bir süre gittikten sonra içlerinden biri kendini başlangıç noktasına götürecek kadar yiyecekle geri döner. Daha sonra ikinci bir kaşif ve diğerleri de geri döner. Yiyeceklerin bir kişiden diğerine aktarılabildiğini ve kaşiflerin tam gün yolculuk yaptığını varsayarsak yolculuk en fazla kaç gün sürmüş olabilir.

**Comment by OmerAkbayrak** [comment]
**Bence cevap: 8, çünkü ilk kaşif 1 gün boyunca başlangıç noktasından ilerler ve 1 gün boyunca da geri döner ve geri dönmeden önce geri kalan 3 günlük yemeğini 4. kaşife verir, 2. kaşif 2 gün gider ve 2 gün geri dönüşü olacağı için geri kalan 1 günlük yemeğini 3. kaşife verir.
3. kaşifin 6 günlük yemeği olacağından 3 gün gider ve 3 gün geri döner. 4. kaşif ise toplamda 8 günlük yemek ile 4 gün gider 4 gün ise geri döner.**
